### DISCUSSION TITLE
error with pandas file

### DISCUSSION THREAD

**Discussion by tahm5** [kyleskom/NBA-Machine-Learning-Sports-Betting#303] [General]
starting the tool with 

$ python3 main.py -xgb -odds=fanduel

I have this error

File "/home/user/NBA-Machine-Learning-Sports-Betting/main.py", line 4, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'

Please if someone guides me thank you

**Comment by BrettWilliams22** [comment]
Hey there I am also having a lot of issues getting this too run perhaps we can help each other. In regards to this issue have you tried running pip3 install pandas and then trying again to compile it?
